### DISCUSSION TITLE
do i need to pay tapo cloud to use this integration?

### DISCUSSION THREAD

**Discussion by purpleendmill** [JurajNyiri/HomeAssistant-Tapo-Control#784] [Discuss]
with the new 6.0.1 release and the tapo 3.8.115 i tried again to install this integration but i get stuck at the cloud password thing
so do i need to pay tapo cloud?
and if not how can i install it?  the password is the correct one i even reseted it to make sure i was no tripping and all i get is 
~Invalid cloud password. Make sure you are entering the password for your cloud account and NOT the password which you created via the camera settings (unless they are the same). You need to enter password which you used with your email when signing into the Tapo app. Ensure you have Third-Party compatibility turned on in Tapo App (Me -> Tapo Lab -> Third-Party Compatibility - On).~
thank you

**Comment by JurajNyiri** [comment]
Both questions are answered in FAQ in readme. 

 No you do not need Tapo care or anything paid. Just follow steps in faq when receiving that error
